package.json

```diff
@@ -66,7 +66,7 @@
     "gatsby-source-git": "^1.1.0",
     "gatsby-source-local-git": "^1.3.0",
     "gatsby-transformer-json": "^5.13.1",
-    "gatsby-transformer-remark": "3.2.0",
+    "gatsby-transformer-remark": "6.13.1",
```

Comment:
See line 117 - Is this ok to be updated?

Reply:
It seems to work alright on build.

Reply:
Actually let me test it again cause I think it might be pumping out an error atm